src/test/java/com/moodybluez/enterprise/EnterpriseApplicationTests.java

```diff
@@ -23,13 +22,12 @@
 class EnterpriseApplicationTests {
 
 	@Autowired
-	private IMoodService moodService;
+	private IMoodDAO moodDAO;
 	@Autowired
-	private IEntryService entryService;
+	private IEntryDAO entryDAO;
 
-	private Mood mood;
-	private Entry entry;
-	private Date date;
+	private Mood mood = new Mood();
+	private Entry entry = new Entry();
```

Comment:
This was a flaw in the test according to me. I am not sure why you did not initialize these and even created another object of same variable type when this was avialable.